exercises/exercisesModels.js

```diff
@@ -0,0 +1,17 @@
+const db = require('../../models/dbConfig');
+
+function findExercises() {
+  return db('exercises');
+}
+
+function findExerciseById(id) {
```

Comment:
If the Pull Request is to view all exercises, this function meant to get one exercise by id should not be included. I think Matt is already working on this.